tracer/src/Datadog.Trace/IAST/Helpers/StringExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="StringExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Datadog.Trace.Iast.Helpers
+{
+    internal static class StringExtensions
```

Comment:
If the text starts with " but not ends, the method should add a " at the end, I assume, something like:

        if (text != null)
        {
            if (!text.StartsWith("\""))
            {
                text = "\"" + text;
            }

            if (!text.EndsWith("\""))
            {
                text = text + "\"";
            }
        }

Reply:
This is meant to be used in the code generator only. But fixed